nitro-protocol/test/expect-succeed.ts

```diff
@@ -0,0 +1,7 @@
+export async function expectSucceed(fn: () => void) {
+  const txResult = (await fn()) as any;
+
+  // As 'requireStateSupported' method is constant (view or pure), if it succeedes, it returns an object/array with returned values
+  // which in this case should be empty
+  expect(txResult.length).toBe(0);
+}
```

Comment:
👍 Good idea to encapsulate this -- if the ethers.js API / behaviour changes, we only need to update this one file.